configs/openfin/manifest-local.json

```diff
@@ -3,7 +3,8 @@
     "startup_app": {
         "name": "ChartIQ Local",
         "url": "http://localhost:3375/finsemble/components/system/serviceManager/serviceManager.html",
-        "uuid": "Finsemble",
+		"uuid": "Finsemble",
+		"requireHandshake": true,
```

Comment:
Inside a comment in FEA it says "by default the check is disabled", so should this flag be false?

Reply:
After discussing with Brad and Ian it was decided to have the value default to false for general compatibility (including if FEA is used without Finsemble).  But we'll always want to have it enabled with Finsemble since it provides one extra check that everything comes up correctly (and puts out notification if it doesn't).   So setting to true is correct.